<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendPost</class>
 <widget class="QWidget" name="sendPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>B Yekan</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>انجام ارسال</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::RightToLeft</enum>
  </property>
  <widget class="QGroupBox" name="sendPostBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>571</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QListWidget" name="postSanaItemsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>شروع   </string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>توقف   </string>
   </property>
  </widget>
  <widget class="QLabel" name="stopwatchLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>زمان: ۰۰:۰۰:۰۰</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
